source/code/providers/Container_HostInventory_Class_Provider.cpp

```diff
@@ -0,0 +1,242 @@
+/* @migen@ */
+#include <MI.h>
+#include "Container_HostInventory_Class_Provider.h"
+
+#include <vector>
+#include <string>
+#include <syslog.h>
+#include <stdio.h>
+#include <uuid/uuid.h>
+#include <cstdlib>
+
+#include "../cjson/cJSON.h"
+#include "../dockerapi/DockerRemoteApi.h"
+#include "../dockerapi/DockerRestHelper.h"
+
+using namespace std;
+
+MI_BEGIN_NAMESPACE
+
+class Guid
```

Comment:
This was copied from elsewhere. Please put it in a header file and reference it from there, here in this code and from any other code that needs it.